content/en/docs/concepts/cluster-administration/networking.md

```diff
@@ -46,8 +46,8 @@ imply any preferential status. The list does not try to be exhaustive.
 * [Amazon AWS VPC CNI plugin](https://github.com/aws/amazon-vpc-cni-k8s)
 * [Amazon Calico for network policy enforcement](https://docs.aws.amazon.com/eks/latest/userguide/calico.html).
```

Comment:
I feel like we should not list every cloud-provider's variant of existing projects.  I don't want to like AKS+calico and GKE+calico and GKE+Cilium and ... here.  This is redundant with Calico, which is listed later.  Remove?

Reply:
Also we have 2 different Azure ones.  I am fine with listing one, but 2 seemns redundant?

Reply:
I agree, removed Amazon+Calico and the link to docs.microsoft.com in favor of their Azure CNI plugin doc on github